parsers/myrddin.c

```diff
@@ -0,0 +1,46 @@
+/*
+ *   Copyright (c) 2016, Ori Bernstein
+ *
+ *   This source code is released for free distribution under the terms of the
+ *   GNU General Public License.
+ *
```

Comment:
Could you write the version of GNU GPL explictly?

http://docs.ctags.io/en/latest/contributions.html#origin-of-changes-and-license